QModManager/Patching/ManifestValidator.cs

```diff
@@ -202,6 +203,7 @@ internal ModStatus FindPatchMethods(QMod qMod)
                                 else
                                 {
                                     qMod.PatchMethods[patch.PatchOrder] = new QModPatchMethod(method, qMod, patch.PatchOrder);
+                                    continue;
```

Comment:
A `continue` at the end of a loop block is redundant.

Reply:
https://discordapp.com/channels/324207629784186882/529634427110948899/706907934164123698